sample/src/main/kotlin/com/natura/android/sample/components/DrawableAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.natura.android.sample.components
+
+import android.content.Context
+import android.support.v7.widget.AppCompatTextView
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import com.natura.android.sample.R
+import java.util.*
```

Comment:
Não seria melhor importar apenas o que for utilizado na classe?

Reply:
verdade. eu sempre esqueco de olhar os imports. Vou corrigir agora.